go/interpreter/lfvm/instructions.go

```diff
@@ -1098,15 +1098,11 @@ func checkSizeOffsetUint64Overflow(offset, size *uint256.Int) error {
 	return nil
 }
 
-func neededMemorySize(c *context, offset, size *uint256.Int) (uint64, error) {
+func checkAndReturnUint64(offset, size *uint256.Int) (uint64, uint64, error) {
```

Comment:
Check what? Returns 2 ints.

Reply:
functioned removed. used another existing one for the checks.